druid/src/command.rs

```diff
@@ -228,37 +229,96 @@ pub mod sys {
 impl Selector {
     /// A selector that does nothing.
     pub const NOOP: Selector = Selector::new("");
+}
 
+impl<T> Selector<T> {
     /// Create a new `Selector` with the given string.
-    pub const fn new(s: &'static str) -> Selector {
-        Selector(s)
+    pub const fn new(s: &'static str) -> Selector<T> {
+        Selector(s, PhantomData)
+    }
+
+    pub(crate) const fn symbol(self) -> SelectorSymbol {
+        self.0
+    }
+}
+
+impl<T: Any> Selector<T> {
+    pub fn carry(self, object: T) -> Command {
```

Comment:
We definitely want `/// Doc lines` for all `pub` items. Let's add one here for `carry`.